scripts/modules/b2b-account/quotes.js

```diff
@@ -369,14 +381,60 @@ define([
                 window.location = '/myaccount/quote/' + quoteId + '/edit';
             }
         },
-        deleteQuote: function () {
-            this.trigger('deleteQuoteView');
+        deleteQuote: function (e, row) {
+            e.stopPropagation();
+            var self=this;
+           var confirmFlag= window.confirm("Do you want to Delete " +row.attributes.name+"?");
+           if(confirmFlag){
+            var quote = new QuoteModels.Quote({
+                id:row.id
+            });
+            return quote.apiModel.removeQuote().then(function (res) {
+                self.refreshGrid();
+            }, function (error) {
+              self.showMessageBar(error.message);
+            });}
         },
-        copyQuote: function () {
-            this.set('copyQuoteView');
+        copyQuote: function (e, row) {
+            var self=this;
+            e.stopPropagation();
+            var quote = new QuoteModels.Quote({
+               id:row.id
+            });
+            return quote.apiModel.copy().then(function (res) {
+                if (isSalesRep) {
+                    window.location = '/selleraccount/quote/' + res.data.id + '/edit';
```

Comment:
save 'res.data.id' into a variable and then use it. name it properly.

Reply:
done